crates/runtime/src/model/eval/runs.rs

```diff
@@ -74,6 +74,7 @@ pub static EVAL_RUNS_TABLE_REFERENCE: LazyLock<TableReference> =
     });
 
 pub static EVAL_RUNS_TABLE_TIME_COLUMN: &str = "created_at";
+pub static EVAL_RUNS_TABLE_TIME_COMPLETED_COLUMN: &str = "completed_at";
```

Comment:
Note: This is a UX/DX change, so technically should be reviewed (although `completed_at` is good).